### ISSUE TITLE
The NE low code prototype can accommodate the updated hours and days fields 

### ISSUE THREAD

**Issue by csprayregen** [pieforproviders/pieforproviders#1148]
## 📋 Summary
We are adding a field to help providers track hours per week (against a maximum). The data/spreadsheet changes for this are in #1146 . To implement this in the low code prototype:

- [ ]  Update dashboard rake task/spreadsheet parser and model to accept both of these
- [ ]  Update api case_list_for_dashboard to return both of these fields
- [ ]  Connect with data team to specify formatting details for this field in the S3 upload spreadsheet and which column numbers to add these fields to

## 💻 Implementation Details

> **NOTE TO ENGINEERS**
> 
> Please ensure that you are running Google Lighthouse and/or WebAIM (Wave) on any UI elements in your pull request to check for accessibility issues.
> 
> If it is out of scope to address those accessibility issues in your pull request (because the issue is in a parent component and needs refactoring to work, for example), **you are responsible for noting those issues in your pull request and writing bug tickets to address the issues - make sure you link the new tickets in your pull request**

**Comment by katelovescode** [comment]
@csprayregen do we need this checklist item:
>  Connect with data team to specify formatting details for this field in the S3 upload spreadsheet and which column numbers to add these fields to

**Comment by csprayregen** [comment]
Yes please @katelovescode ! You can let them know the format in #1149

**Comment by katelovescode** [comment]
@rebeldroid12 turns out this is done already 😭 sorry!!!
